bokeh/compat/mplexporter/exporter.py

```diff
@@ -189,7 +189,7 @@ def draw_line(self, ax, line, force_trans=None):
         if (markerstyle['marker'] in ['None', 'none', None]
                 or markerstyle['markerpath'][0].size == 0):
             markerstyle = None
-        label = line.get_label()
+        label = line._get_label()
```

Comment:
Seems I must have misapplied a refactor on that file. Just fixed it and pushed the change. I checked the remaining files to make sure nothing else was like that